schema.graphql

```diff
@@ -263,6 +267,20 @@ enum EventOrderByInput {
   title_DESC
 }
 
+type EventProject {
```

Comment:
@palisadoes wasn't this removed?

Reply:
Yes it should have been

Reply:
@duplixx 

1. We removed Projects and Tasks from the API a few PRs ago.  Your starting code is out of date
    - https://github.com/PalisadoesFoundation/talawa-api/pull/1627
1. This is going to be replaced with Action Items.
    - https://github.com/PalisadoesFoundation/talawa-api/pull/1648
1. Remove this Type and all references to it